cli/build.go

```diff
@@ -117,7 +119,7 @@ func (b *BuildCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{})
 		// Setup dependencies
 		containers := target.Dependencies.Containers
 
-		if len(containers) > 0 {
+		if len(containers) > 0 && !b.NoSideContainer {
```

Comment:
If we want to ignore side containers completly... That was what gave that Kotinos bug today.